includes/class-apple-news.php

```diff
@@ -79,6 +79,32 @@ public static function get_filename( $path ) {
 	}
 
 	/**
+	 * Validate settings and see if any updates need to be performed.
+	 *
+	 * @param array $wp_settings An array of settings loaded from WP options.
+	 *
+	 * @access public
+	 * @return array The modified settings array.
+	 */
+	public function validate_settings( $wp_settings ) {
+		// If this option doesn't exist, either the site has never installed
+		// this plugin or they may be using an old version with individual
+		// options. To be safe, attempt to migrate values. This will happen only
+		// once.
+		if ( false === $wp_settings ) {
```

Comment:
Nah it's because we changed the option name at one point. It would return false here if not set and then we can try to migrate settings.